Exception caught @00:45:10 11.13.12
    Message: An error occurred while inserting record data
    SQL: INSERT INTO game (player_1, player_2) VALUES (?,?)
    File: /Users/jdesulme/Documents/Workspace/Connect4/BizDataLayer/challengeData.php
    Line: 140
    Trace: #0 /Users/jdesulme/Documents/Workspace/Connect4/svcLayer/challenge/challengeUtil.php(70): createNewGame('12', '13')
#1 [internal function]: updateChallenge('12|13|A', '129.21.118.201', 'ec3ba107beaafdb...')
#2 /Users/jdesulme/Documents/Workspace/Connect4/mid.php(17): call_user_func('updateChallenge', '12|13|A', '129.21.118.201', 'ec3ba107beaafdb...')
#3 {main}


